apps/android/README.md

```diff
@@ -23,6 +24,16 @@ yarn build
 yarn android
 ```
 
+M1 Macs:
```

Comment:
Rather than have have 3 different additions, I think it might make sense to just have one section like "If you are using an Apple Silicon Mac" and condense the 3 sections into one.